Osiris/SDK/NetworkChannel.h

```diff
@@ -4,8 +4,9 @@
 
 class NetworkChannel {
 public:
-    VIRTUAL_METHOD(float, getLatency, 9, (int flow), (this, flow))
+    VIRTUAL_METHOD(float, getLatency, IS_WIN32() ? 9 : 8, (int flow), (this, flow))
+
+    std::byte pad[WIN32_LINUX(43, 63)];
```

Comment:
`NetworkChannel::chokedPackets` is 4-byte aligned, so this isn't correct.

Reply:
Sorry for my mistakes but when I tested it on Linux it at least seems to be working, Increasing / decreasing the choked packets did feel correct (More lag as I increased, and less when decreased)

My understanding was that `std::byte pad` had a size of 1 + (43 / 63) which makes `NetworkChannel::chokedPackets` 44 (0x002C) on Windows and 64 (0x0040) on Linux?

Linux Reference - https://github.com/LWSS/Fuzion/blob/master/src/SDK/INetChannel.h#L177

Windows Reference - https://github.com/AllusiveWheat/skeet.pw/blob/cb77743dad6abec5acc1fbbdc1a1f7d38c0165a3/sdk/interfaces/IVEngineClient.hpp#L351

Very new to this stuff, would appreciate if you could explain a little.

Thanks.

Reply:
I think this could use `__builtin_frame_address()` for Linux and `_AddressOfReturnAddress()` for Windows right?

In my opinion it simplifies things.

Reply:
It works due to struct alignment which aligns `NetworkChannel::chokedPackets` to 4-byte offset but it's misleading to have 43/63 bytes padding. When you want your field to be at **std::uintptr_t(this) + 0x40** that means you need 64-byte padding.